components/CustomerInfo/styles.js

```diff
@@ -25,11 +33,52 @@ const CustomerInfoButtonContainer = styled.section`
   display: flex;
   margin-top: 30px;
   gap: 50px;
+  margin-bottom: 130px;
+`;
+
+const InfoParagraph = styled.p`
+  background-color: ${(props) => props.theme.fifth};
+  padding: 20px;
+  width: 80%;
+  border-radius: 10px;
 `;
+
+const InfoParagraphContainer = styled.section`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+`;
+
+const EditButton = styled.button`
+  background-color: ${(props) => props.theme.secondary};
+  color: ${(props) => props.theme.third};
+  padding: 10px 20px;
+  border-radius: 10px;
+  border: none;
+`;
+
+const InfoModalStyle = {
+  content: {
+    top: "50%",
+    left: "50%",
+    right: "auto",
+    bottom: "auto",
+    marginRight: "-50%",
```

Comment:
You don't need the `marginRight: "-50%"` property here. The `transform: "translate(-50%, -50%)"` property already centers the modal horizontally and vertically using the top, left, and transform properties.
```suggestion

```